src/shared/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -499,20 +499,22 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
               />
               <CustomControls isFullScreen={isFullScreen} isEnded={playerState === 'ended'}>
                 <PlayControl isLoading={playerState === 'loading'}>
-                  <PlayButton
-                    isEnded={playerState === 'ended'}
-                    onClick={handlePlayPause}
-                    tooltipText={isPlaying ? 'Pause (k)' : playerState === 'ended' ? 'Play again (k)' : 'Play (k)'}
-                    tooltipPosition="left"
-                  >
-                    {playerState === 'ended' ? (
-                      <SvgPlayerRestart />
-                    ) : isPlaying ? (
-                      <SvgPlayerPause />
-                    ) : (
-                      <SvgPlayerPlay />
-                    )}
-                  </PlayButton>
+                  {!showBigPlayButton && (
```

Comment:
This will remove play button on desktop view.
![Zrzut ekranu 2021-11-15 o 12 47 17](https://user-images.githubusercontent.com/51168865/141777250-d8ef8787-5d62-495c-a437-be1ec1f23dac.png)


@toiletgranny what do you think about it?

Reply:
Thanks for including me in this! 

Frankly, I would recommend having the small play button visible and active in the above case scenario. This might be useful for users that have already developed a "muscle memory" for playing/pausing the video in the bottom left corner of the video player.

To have the big play button in the middle, as an addition, is ok for me, by the way.